lib/block-supports/layout.php

```diff
@@ -53,46 +53,24 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	$content_size = isset( $used_layout['contentSize'] ) ? $used_layout['contentSize'] : null;
 	$wide_size    = isset( $used_layout['wideSize'] ) ? $used_layout['wideSize'] : null;
 
-	$all_max_width_value       = $content_size ? $content_size : $wide_size;
-	$all_max_width_declaration = "max-width: $all_max_width_value";
-
-	$wide_max_width_value       = $wide_size ? $wide_size : $content_size;
-	$wide_max_width_declaration = "max-width: $wide_max_width_value";
+	$all_max_width_value  = $content_size ? $content_size : $wide_size;
+	$wide_max_width_value = $wide_size ? $wide_size : $content_size;
 
 	$style = '';
 	if ( $content_size || $wide_size ) {
-		ob_start();
-		?>
-			<?php echo '.wp-container-' . $id; ?> > * {
-				<?php echo esc_html( safecss_filter_attr( $all_max_width_declaration ) ); ?>;
-				margin-left: auto !important;
-				margin-right: auto !important;
-			}
-
-			<?php echo '.wp-container-' . $id; ?> > .alignwide {
-				<?php echo esc_html( safecss_filter_attr( $wide_max_width_declaration ) ); ?>;
-			}
-
-			<?php echo '.wp-container-' . $id; ?> .alignfull {
-				max-width: none;
-			}
-		<?php
-		$style = ob_get_clean();
-	}
+		$style  = ".wp-container-$id > * {";
+		$style .= 'max-width: ' . esc_html( $all_max_width_value ) . ';';
+		$style .= 'margin-left: auto !important;';
+		$style .= 'margin-right: auto !important;';
+		$style .= '}';
 
-	ob_start();
-	?>
-		<?php echo '.wp-container-' . $id; ?> .alignleft {
-			float: left;
-			margin-right: 2em;
-		}
+		$style .= ".wp-container-$id > .alignwide { max-width: " . esc_html( $wide_max_width_value ) . ';}';
```

Comment:
Just nitpicking, but why not use `sprintf` here?  It's a common pattern in core for string concat.

Reply:
The fact that it's a common pattern doesn't necessarily mean it's a good one :sweat_smile: 
`sprintf` in core is a bit overused, and it makes sense to use it when the string is to be translated and contains vars. But when `sprintf` is used to concat simple, non-translatable strings, calling a function to concat is a bit of an overkill

Reply:
> The fact that it's a common pattern doesn't necessarily mean it's a good one
😂

Makes sense.